openmetadata-service/src/main/java/org/openmetadata/service/resources/kpi/KpiResource.java

```diff
@@ -387,8 +386,7 @@ public Response restoreKpi(
   public Response addKpiResult(
       @Context UriInfo uriInfo,
       @Context SecurityContext securityContext,
-      @Encoded @Parameter(description = "fqn of the kpi", schema = @Schema(type = "string")) @PathParam("fqn")
-          String fqn,
+      @Parameter(description = "fqn of the kpi", schema = @Schema(type = "string")) @PathParam("fqn") String fqn,
```

Comment:
@TeddyCr, why should fqn be encoded here? It is used in findEntityByName(fqn) all the time for other entities where the fqn is not encoded. I also believe this does not have any upgrade impact. Please confirm.

Reply:
@sureshms yes , it need not be encoded for the KPI